modules/lang-painless/src/main/java/org/opensearch/painless/antlr/package-info.java

```diff
@@ -26,7 +26,7 @@
  */
 
 /**
- * Lexer, parser, and tree {@link Walker} responsible for turning the code
+ * Lexer, parser, and tree walker responsible for turning the code
```

Comment:
fyi: these unrelated javadocs changes are just so that the module would compile in eclipse for me (it is picky by default about such stuff). I tried to avoid touching src/java in this PR, and I know eclipse is unsupported, but I use it for refactoring such as "rename method".